src/context/volume-context.js

```diff
@@ -0,0 +1,82 @@
+import {createContext, useContext, useReducer} from "react";
+
+const VolumeContext = createContext();
+
+// Custom component to provide the Volume context
+function VolumeProvider(props) {
+  const [state, dispatch] = useReducer(volumeReducer, {
+    toggles: {
+      measurement: 0,
+      season: 0, 
+      tide: 0,
+    },
```

Comment:
Are `0` and `1` the best types here? Would a boolean serve you better as they can only be `0` or `1`? Do you foresee things expanding in the future?

Reply:
Mary made a similar point. I changed them to strings - `low` and `high`, `winter` and `summer`, etc